src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1796,18 +1796,17 @@ export const ComputeModalBase = ({
                 side: 'bottom'
               }, [renderPersistentDiskType(id)]) : renderPersistentDiskType(id),
             h(div, [
-              label({ htmlFor: id, style: computeStyles.label }, ['Disk Size (GB)']),
-              div({ style: { marginTop: '0.5rem' } }, [
+              label({ htmlFor: id, style: computeStyles.label }, [
+                'Disk Size (GB)',
```

Comment:
Nesting this causes some spacing issues. In this case, I would recommend you stick with the id approach (you have the IdContainer still anyway).

Before:
![image](https://user-images.githubusercontent.com/484484/215588519-592b5bd2-490e-4c44-b414-b47f99b71e72.png)

After:
![image](https://user-images.githubusercontent.com/484484/215588478-9b51ede5-8c3a-4f6a-ad76-8ed3e70829ba.png)